data/data/baremetal/bootstrap/outputs.tf

```
@@ -0,0 +1,3 @@
+output "bootstrap_ip" {
+  value = length(libvirt_domain.bootstrap.*.network_interface.0.addresses[0]) > 0 ? libvirt_domain.bootstrap.*.network_interface.0.addresses[0] : null
```

Why do we need to force null here?
```suggestion
  value = libvirt_domain.bootstrap.*.network_interface.0.addresses[0]
```